internal/file/file.go

```diff
@@ -94,3 +117,15 @@ func PrepareContent(path string, mediaType string, dgstStr string, size int64) (
 		Size:      actualSize,
 	}, file, nil
 }
+
+// ParseMediaType parses the media type field of bytes content in json format.
+func ParseMediaType(content []byte) (string, error) {
+	var manifest map[string]interface{}
+	if err := json.Unmarshal(content, &manifest); err != nil {
+		return "", errors.New("not a valid json file")
+	}
+	if manifest["mediaType"] == nil {
+		return "", errors.New("media type is not recognized")
+	}
+	return fmt.Sprint(manifest["mediaType"]), nil
+}
```

Comment:
Here's a trick in golang.
```suggestion
// ParseMediaType parses the media type field of bytes content in json format.
func ParseMediaType(content []byte) (string, error) {
	var manifest struct {
		MediaType string `json:"mediaType"`
	}
	if err := json.Unmarshal(content, &manifest); err != nil {
		return "", errors.New("not a valid json file")
	}
	if manifest.MediaType == "" {
		return "", errors.New("media type is not recognized")
	}
	return manifest.MediaType, nil
}
```